src/quo2/screens/avatars/channel_avatar.cljs

```diff
@@ -28,14 +28,14 @@
                              :emoji                  "🍑"
                              :emoji-background-color :gray})]
     (fn []
-      [rn/view {:margin-bottom 50
-                :padding       16}
-       [rn/view {:flex 1}
-        [preview/customizer state descriptor]]
-       [rn/view {:padding-vertical 60
-                 :flex-direction   :row
-                 :justify-content  :center}
-        [quo2/channel-avatar @state]]])))
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 150}
```

Comment:
`:padding-bottom 150` is used to make sure there is always enough touchable area for closing the keyboard, even if there are only a few inputs available. [example](https://user-images.githubusercontent.com/17097240/191633748-da007efa-8521-488a-84ac-b387a5486e03.png)